src/components/CallView/shared/VideoBottomBar.vue

```diff
@@ -278,17 +298,28 @@ export default {
 	position: absolute;
 	bottom: 0;
 	width: 100%;
-	padding: 0 12px 8px 12px;
+	padding: 0 12px 4px 12px;
 	z-index: 1;
 
 	&--big {
 		justify-content: center;
+		margin: 0 var(--default-clickable-area); // grid collapse button
+		width: calc(100% - var(--default-clickable-area) * 2);
 		& .bottom-bar {
 			width: unset;
+			padding: 0 4px;
 		}
+
 		& .participant-name {
 			margin-right: 0;
 		}
+
+		&:hover {
```

Comment:
Feels a bit counter-intuitive, so the whole line is now freezing the model, but feedback is only on smaller part.
Maybe, we keep `@mouseover` on the bottom-bar, as it was initially, and increase it's padding?

wrapper | bottombar
-- | --
![image](https://github.com/user-attachments/assets/6887d770-3b51-42bf-89d8-ca33f7cd2f4d) | ![image](https://github.com/user-attachments/assets/d92cf42a-e3e9-4c5d-8b4a-76e3baabb7bb)
![image](https://github.com/user-attachments/assets/afde6ccc-1e8a-4b80-ba66-a579bf6efb35) | ![image](https://github.com/user-attachments/assets/5d54f310-a98d-4d5e-b422-2ff00249d399)

Reply:
It was indented, so to have more area to apply freeze as you mentioned it's a small area initially and user can miss it. I will revert to the initial state.